keyboards/centromere/keymaps/mattly/keymap.c

```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
+#include "mattly.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  // I apparently soldered in my switches on the wrong sides of the boards, so this is mirrored
+
+  [_BASE_MAC] = LAYOUT_split_3x6_3( \
```

Comment:
```suggestion
  [_BASE_MAC] = LAYOUT_split_3x6_3(
```

Reply:
ah, this is leftover from the `default` keymap which I had copied.